src/components/input-date-picker/input-date-picker.scss

```diff
@@ -91,8 +91,7 @@
 .menu-container {
   @include floatingUIContainer();
   @include floatingUIWrapper();
-  @apply invisible
-    pointer-events-none;
+  @apply invisible;
```

Comment:
@jcfranco since this is just a CSS fix, we can probably create something for @AdelheidF to use to apply to a specific instance and remove this style.

Something like...

```
var style = document.createElement( 'style' )
style.innerHTML = '.menu-container { pointer-events: inherit !important; }'
datePicker.shadowRoot.appendChild( style )
```

Reply:
This would be good for a quick patch fix for sure.

Reply:
maybe add the workaround to the issue

Reply:
Yeah I can give that a shot.

Reply:
This workaround can be used:

```
 <script>
    (async () => {
      await customElements.whenDefined("calcite-input-date-picker");
      const inputDatePicker = document.querySelector("calcite-input-date-picker");
      await inputDatePicker.componentOnReady();

      var style = document.createElement("style");
      style.innerHTML = ".menu-container { pointer-events: inherit !important; }";

      inputDatePicker.shadowRoot.appendChild(style);
    })();
  </script>
```

Reply:
@AdelheidF can you try this out?

I'm guessing you'll need to do something like this...

```jsx
<calcite-input-date-picker
ref={(inputDatePicker: HTMLCalciteInputDatePickerElement) => {
   const style = document.createElement("style");
   style.innerHTML = ".menu-container { pointer-events: inherit !important; }";
   inputDatePicker.shadowRoot.appendChild(style)
}}>
```

Reply:
works for me. Thanks!